src/pages/accountSetup/AccountSetupPage.jsx

```diff
@@ -0,0 +1,378 @@
+                      </Typography>
+                    </Grid>
+                    <Grid item xs={12} sm={6}>
+                      <CustomSelect
+                        options={pointSizeOptions}
+                        aria-describedby="main-text-point-size-helper-text"
+                        id="main-text-point-size-helper-text"
+                        formik={formik}
+                        fieldName="mainTextPointSize"
+                        helperText={
+                          <StyledFormHelperText id="main-text-font-helper-text">
+                            {formik.touched.mainTextPointSize &&
+                              formik.errors.mainTextPointSize}
+                          </StyledFormHelperText>
+                        }
+                      />
+                      {/* <StyledFormHelperText id="main-text-point-helper-text">
+                          {formik.touched.mainTextPointSize &&
+                            formik.errors.mainTextPointSize}
+                        </StyledFormHelperText> */}
+                    </Grid>
+                    <Grid item xs={12} sm={6}>
+                      <Typography variant="subtitle1" sx={{ fontSize: 20 }}>
+                        Your System Name
+                      </Typography>
+                    </Grid>
+                    <Grid item xs={12} sm={6}>
+                      <FormControl
+                        fullWidth
+                        error={
+                          formik.touched.systemName &&
+                          Boolean(formik.errors.systemName)
+                        }
+                      >
+                        <CustomTextField
+                          placeholder="Enter"
+                          aria-describedby="system-name-helper-text"
+                          fieldName={"systemName"}
+                          formik={formik}
+                        ></CustomTextField>
+
+                        <StyledFormHelperText id="system-name-helper-text">
+                          {formik.touched.systemName &&
+                            formik.errors.systemName}
+                        </StyledFormHelperText>
+                      </FormControl>
+                    </Grid>
+                    <Grid item xs={12} sm={6}>
+                      <Typography variant="subtitle1" sx={{ fontSize: 20 }}>
+                        Two-Factor Authentication
+                      </Typography>
+                    </Grid>
+                    <Grid item xs={12} sm={6}>
+                      <StyledCheckbox
+                        name="twoFactAuth"
+                        // do not remove double negation or else your see an uncontrolled to controlled switchBase error
+                        // refer to https://stackoverflow.com/questions/69259429/material-ui-a-component-is-changing-the-uncontrolled-checked-state-of-switchbas
+                        checked={!!formik.values.twoFactAuth}
```

Comment:
This might actually not be needed given authentication is handle by google... unless @MultipliedByOne had another idea about this checkbox